target/linux/ath79/dts/ar9344_mikrotik_routerboard-sxt-5nd-r2.dts

```diff
@@ -6,3 +6,11 @@
 	compatible = "mikrotik,routerboard-sxt-5nd-r2", "qca,ar9344";
 	model = "MikroTik RouterBOARD SXT 5nD r2 (SXT Lite5)";
 };
+
+&routerboot {
+	bootloader2: partition@10000 {
```

Comment:
Is the bootloader2: DT label needed for something I don't see, or can it be dropped?

Reply:
Dropped, thanks.

Reply:
Don't do this. Don't split partitions between dtsi and dts. This is going to be a maintenance nightmare.
Why do you need to do this to begin with? The 64kB devices do have a secondary bootloader.